rocm-device-libs/PKGBUILD

```diff
@@ -0,0 +1,23 @@
+# Maintainer: Ranieri Althoff <ranisalt+aur at gmail dot com>
+
+pkgname=rocm-device-libs
+pkgver=3.3.0
+pkgrel=1
```

Comment:
I believe the pkgver should be updated here.

Reply:
Updated to what? It's on v3.3.0